src/pages/bradford/severe-weather-accommodation/index.hbs

```diff
@@ -31,12 +31,12 @@ page: swep
     <div class="block__content">
       <p>The Cold Weather Provision (CWP) is a multi-agency partnership that provides emergency bed spaces in Bradford to rough sleepers on freezing winter nights, commissioned by Bradford Council.</p>
       <p>The service operates between 1st October and 31st April and opens when the predicted temperature for the upcoming evening is due to fall to zero or below and provides an emergency bed to clients in a communal room or with another housing provider.</p>
-      <p>Referrals to the provision can be made in person at Discovery House, 133 Barkerend Road, BD3 9AU, over the phone <a href="tel:01274 309165">01274 309165</a>, through Bradford Council’s Housing Options Service and other advisory housing and support services in the district.</p>
+      <p>Referrals to the provision can be made in person at <a href="http://www.disc-vol.org.uk/projects/bradford-no-second-night-out-housing-support/">Discovery House</a>, <a href="https://www.google.com/maps/place/133+Barkerend+Rd,+Bradford+BD3+9AU/@53.7971133,-1.7443232,17z/data=!3m1!4b1!4m5!3m4!1s0x487be14478e6aef9:0x1524f93f478d4140!8m2!3d53.7971133!4d-1.7421345">133 Barkerend Road, BD3 9AU</a>, over the phone <a href="tel:01274 309165">01274 309165</a>, through Bradford Council’s Housing Options Service and <a href="/find-help/accommodation/listing/">other advisory housing and support services</a> in the district.</p>
```

Comment:
should the advisory housing and support services link be going to the accommodation advice (https://streetsupport.net/find-help/support/?location=my-location&range=10000&sub-category=housing) or the accommodation listing?

Reply:
He said "...there does need to be a link from "housing and support services" to the accomodation section of the website". I assumed he meant the accommodation listing page but I'll need to check with him next week if not right - or release to UAT and we check with him. The direction was a bit vague.

Reply:
it sounds like its support services, rather than specific places to stay